pkg/uzlib/doc.txt

```diff
@@ -4,6 +4,10 @@
  * @ingroup  sys_compression
  * @brief    Deflate/Zlib-compatible LZ77 compression/decompression library
  *
+ * @warning During compression, uzlib allocates memory on the heap without freeing it.
+ *          Do not use this package in production code!
+ *          See https://github.com/pfalcon/uzlib/issues/41#issuecomment-1907807048
```

Comment:
That sounds a lot scarier than it is, AFAIU the API is just a bit bad in that it allocates the memory on it's own and int a bit of an opaque location, but it's the `outbuf` that then contains the result of the compression.

So it should be enough if we free `free(comp.out.outbuf)` once we are done with it, right?

Reply:
Yes, and it should be possible to fix this with a simple git patch or upstream PR.